### ISSUE TITLE
File structure for the XML files

### ISSUE THREAD

**Issue by DavidHaslam** [lb42/KJV_1611#6]
In view of the way GitHub throws a "wobbler" for a folder containing more than 1000 files,

> Sorry, we had to truncate this directory to 1,000 files. 361 entries were omitted from the list. 

I wonder if it's worth considering changing the folder/file structure to have a **books** folder with 79 folders, each named after a book and containing the set of chapter files for that book ?

Yes - it's a big change, but it would mean that any visitor could more readily access the XML files for the books with names later in the alphabet than `Matthew_19.xml`.

**Comment by lb42** [comment]
Yes, I had indeed thought of doing just that, or maybe dividing it into OT NT and AP, but it's rather destabilizing.  I'll think about it some more, since I agree that it would be nice to be able to access individual chapters from the github index page, You can of course access them by cloning them to your desktop and reorganizing them as you wish.

**Comment by DavidHaslam** [comment]
Which is what I did, yet I was thinking about future visitors that might merely wish to examine [say] Matthew 28 or [say] Romans 5.

It's not urgent, being merely a rearrangement, and potentially messy to implement.
